docs/reference/method/MongoDBClient-createClientEncryption.txt

```diff
@@ -0,0 +1,51 @@
+=========================================
+MongoDB\\Client::createClientEncryption()
+=========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Client::createClientEncryption()
+
+   Returns a ClientEncryption object for manual encryption and decryption of
```

Comment:
I think it'd be reasonable to use

```
:php:`MongoDB\\Driver\\ClientEncryption <class.mongodb-driver-clientencryption>`
```

here just to provide users another direct link to the actual API they'll end up using. I've noted that it's also in the return section below.